openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,20 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' in data:
+            if 'load' in data['resources']:
+                self.resources_metric_write(
+                    pk, ct, key='load', value=100 * data['resources']['load'][1]
+                )
```

Comment:
We should save the three values in `data['resources']['load']` with keys `load_1`, `load_5` and `load_15`. There is no need to save 100 * value

Reply:
There is a need as I mentioned above threshold value can only be an integer, hence `*100`. @PabloCastellano , saving the three values doesn't matter I think (I might be wrong) since the device status gets updated every 5 mins and hence I had used that value only specifically. My attempt was to store only necessary info on influxdb.

Reply:
It doesn't matter if the data is collected every 5min. You may want to check the history of cpu load. Don't be afraid of saving useful data :-)

Reply:
Why don't you just save data['resources']['load'] (the full list, without accessing the indexes)?

Reply:
`InfluxDB` doesn't allow it, I tried it.